cmake/MacOSXBundleInfo.plist.in

```diff
@@ -44,5 +44,28 @@
     <string>${MACOSX_SPARKLE_UPDATE_PUBLIC_KEY}</string>
     <key>SUFeedURL</key>
     <string>${MACOSX_SPARKLE_UPDATE_FEED_URL}</string>
+    <key>CFBundleDocumentTypes</key>
+    <array>
+        <dict>
+            <key>CFBundleTypeExtensions</key>
+			<array>
+				<string>zip</string>
+                <string>mrpack</string>
```

Comment:
Indentation here is a little weird

Reply:
these commits are part of #229 so theses should probably be addressed there.